lib/src/components/chat/ds_message_bubble.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:blip_ds/src/theme/systems_colors.dart';
+import 'package:flutter/material.dart';
+
+class DSMessageBubble extends StatelessWidget {
+  final bool alignRight;
```

Comment:
Here I think is better to create an "align" named param with an enum with align options, like Aligment.left, Aligment.right, Aligment.center. We may use this component for cards that must be centered in screen, like a "Ticket closed by client inactivity".